boards/riscv/hpm6750evkmini/doc/index.rst

```diff
@@ -0,0 +1,109 @@
+.. _hpm6750evkmini:
+
+HPMicro HPM6750EVKMINI
+######################
+
+Overview
+********
+
+The HPM6750 is a dual-core flashless MCU running 816Mhz.
+It has a 2MB continuous on-chip ram.
+Also, it provides various memory interfaces, including SDRAM, Quad SPI NOR Flash, SD/eMMC.
+
+The figure shows the PCB is HPM6750EVKmini platform.
+
+.. image:: img/hpm6750evkmini.png
+     :align: center
+     :alt: HPM6750EVKMINI
+
+Hardware
+********
+
+The HPM6750EVKMINI platform integrates 2 cores 32-bit 816MHz RISC-V CPUs, DSP,
```

Comment:
Nitpick: `HPM6750EVKMINI` was previously spelled as `HPM6750EVKmini`. This documentation could use some consistency here.